views/about.ejs

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
```

Comment:
All the views should just be:

<%- include("partials/header") -%>
<%- include("partials/navbar") -%>

</html>